.github/workflows/build-test.yml

```diff
@@ -9,15 +9,21 @@ on:
    workflow_dispatch:
 jobs:
   docker:
-    name: "${{ matrix.DEPLOYMENT_NAME }}  USE_FROZEN=${{ matrix.USE_FROZEN }} -- Image Build, Test"
+    name: "${{ matrix.DEPLOYMENT_NAME }}  USE_FROZEN=${{ matrix.USE_FROZEN }} ${{ matrix.CAL_VERSION }} -- Image Build, Test"
```

Comment:
Probably a dumb question, but what is matrix here?

Reply:
if you look at matrix a few lines under, it basically will run one job for each permutation of all the different matrices. so it'll run name: roman , use_frozen = 0 then name: roman, use_frozen = 1, then name: tike, use_frozen = 0 etc.

Reply:
am I correct in assuming that you removed chilly build from the matrix to cut down on it running so many extra jobs?

Reply:
Yeah,  and frozen.   We need to switch to a mode where we only freeze/chill in the post-merge build tests;  so developers can be hands-off and mostly forget about freezing personally,  and we don't pay for 9 builds on every PR update.   CodeBuild for roman will be really expensive unless we're careful.  But running automated tests on the hub should mitigate that some,  if we can work that out.